### PR TITLE
JSON file of GNOME updated to new syntax

### PR DESCRIPTION
**Pull request by sagarhani**

### CODE REVIEW

**Comment by MrChrisW** [comment]
@sagarhani Looks like you've included your Gmail CheatSheet (#1232) here? Can you remove it from this PR.

**Comment by sagarhani** [comment]
@MrChrisW Now ?

**Comment by MrChrisW** [comment]
@sagarhani :+1:

**Comment by sagarhani** [comment]
This will fix #1250 completely. :+1: 
Now, All IAs have new syntax of JSON file.

**Comment by moollaza** [comment]
@sagarhani this is great, thanks!

There's still a few other changes that need to be made, and I'm thinking we should update the docs to mention Sentence Case, plus Capitals should be used:

```
//bad
[right click], [a]

//good
[Right Click], [A]
```

Also it looks like some of these have alternatives and sub-alternatives that should be updated accordingly

**Comment by moollaza** [comment]
Would you like to tackle those changes as well? I can merge this if you'd like.

---

BTW, for changes like these that update the look of an IA, it realy helps if you take a screenshot so we can verify it looks correct a little quicker :+1:

**Comment by sagarhani** [comment]
@moollaza As of now you can merge this. :+1: 
It would be good if the docs are updated early. :ok_hand: 

After that you can create an issues so that people look at docs and work on.

--
I tested those IAs. It completely looks good. :+1: 


vim-cheat-sheet

gimp-cheat-sheat

blender-cheat-sheet

File: share/goodie/cheat_sheets/blender.json [hunk 1]
```diff
@@ -300,54 +300,54 @@
...
-            "key": "[Alt] + [f]"
+            "key": "[Alt] [f]"
         }, {
             "val": "Beauty fill",
-            "key": "[Shift] + [Alt] + [f]"
+            "key": "[Shift] [Alt] [f]"
         }, {
             "val": "Add subdivision level",
-            "key": "[Ctrl] + [1] / [2] / [3] / [4]"
+            "key": "[Ctrl] [1] / [2] / [3] / [4]"
```
Comment (moollaza) [inline share/goodie/cheat_sheets/blender.json]: These sub-alternatives should be wrapped in parenthesis:

```
[Ctrl] ([1] / [2] / [3] / [4])
```

File: share/goodie/cheat_sheets/blender.json [hunk 2]
```diff
@@ -223,56 +223,56 @@
             "key": "v"
         }, {
             "val": "Weight paint mode",
-            "key": "[Ctrl] + [Tab]"
+            "key": "[Ctrl] [Tab]"
         }, {
             "val": "Switching workspace",
-            "key": "[Ctrl] + [Left arrow] or [Ctrl] + [Right arrow]"
+            "key": "[Ctrl] [Left arrow] or [Ctrl] [Right arrow]"
```
Comment (moollaza) [inline share/goodie/cheat_sheets/blender.json]: This should be `[Ctrl] ([←]/[→])`

File: share/goodie/cheat_sheets/gimp.json [hunk 3]
```diff
@@ -44,47 +44,47 @@
...
         ],
         "Layers": [
             {
-                "key": "[PgUp] or [Ctrl] + [Tab]",
+                "key": "[PgUp] or [Ctrl] [Tab]",
                 "val": "Select the layer above"
             },
             {
-                "key": "[PgDn] or [Shift] + [Ctrl] + [Tab]",
+                "key": "[PgDn] or [Shift] [Ctrl] [Tab]",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/gimp.json]: These alternatives should be 2 separate entires:

```
{
  "key": "[PgDn]",
   "val": "Select the layer below"
},
{
   "key":  "[Shift] [Ctrl] [Tab]",
   "val": "Select the layer below (alternative)"
}
```

**Comment by moollaza** [comment]
@sagarhani sorry to nitpick, but there's still some minor changes that need to be made regarding the alternatives and other things like arrows.

>It would be good if the docs are updated early. :ok_hand:
After that you can create an issues so that people look at docs and work on.

I'm not too sure what you mean? The docs were updated before I made the issue that there's been syntax changes. Perhaps the changes weren't too clear?

**Comment by sagarhani** [comment]
@moollaza I meant these change of syntax must also be updated on the docs as early as possible.
```
//bad
[right click], [a]

//good
[Right Click], [A]
```
This has to be clearly explained in the docs on how to use it.
Also what if there are **case sensitive** ```key``` like these ? 

vim-1

vim-2

File: share/goodie/cheat_sheets/gnome-keyboard.json [hunk 4]
```diff
@@ -10,66 +10,69 @@
...
-            "key": "[Ctrl] + [V] "
+            "key": "[Ctrl] [V] "
         }, {
             "val": "Undo the last action",
-            "key": "[Ctrl] + [Z] "
+            "key": "[Ctrl] [Z] "
         }],
          "Screen Capturing": [{
             "val": "Take a screenshot",
             "key": "[Prnt Scrn]"
```
Comment (codenirvana) [inline share/goodie/cheat_sheets/gnome-keyboard.json]: @sagarhani ```[Prnt Scrn]``` Please fix this too.
According to @moollaza  Single keys shouldn't be wrapped in brackets!
Thanks for updating all the cheat sheets.

**Comment by sagarhani** [comment]
@codenirvana  Thanks for pointing these. :smiley: 
I fixed it. :+1:

**Comment by moollaza** [comment]
>@moollaza I meant these change of syntax must also be updated on the docs as early as possible...This has to be clearly explained in the docs on how to use it.

Can you explain what about it is unclear? The idea is that uppercase letters should always be used. The casing shouldn't imply anything. Meaning if you need to press `[Shift]` to get the Uppercase form of the letter, then that should be part of the information, and explicitly listed as one of the keys that needs to be pressed. This ensures we remove any ambiguity in knowing exactly what keys should be pressed.

>Also what if there are case sensitive key like these ? 

As explained above, there should be no ambiguity.

This would be:

```
[I]
[Shift] [I]
[A]
[Shift] [A]
...
```

Does that make sense?

**Comment by moollaza** [comment]
Closing. These changes have been made+merged elsewhere

**Pull request closed without merging**